File: api/src/Quiniela.Api/Entities/Match.cs
using Quiniela.Api.Enums;

namespace Quiniela.Api.Entities;

public class Match
{
    public Guid Id { get; set; }
    public Guid RoundId { get; set; }
    public string HomeTeam { get; set; } = string.Empty;
    public string AwayTeam { get; set; } = string.Empty;
    public DateTime StartDateTime { get; set; }
    public MatchResult Result { get; set; }
    public bool IsLocked { get; set; }
}

File: src/app/features/rounds/models/match.ts
export enum MatchResult {
  Pending = 0,
  HomeWin = 1,
  AwayWin = 2,
  Draw = 3,
}

export interface Match {
  id: string;
  roundId: string;
  homeTeam: string;
  awayTeam: string;
  startDateTime: string;
  result: MatchResult;
  isLocked: boolean;
}
